docs/tutorial.md

```diff
@@ -244,4 +291,15 @@ Alternatively, you can run the following alias:
 
 ```bash
 ocsdk-delete-operator
-```
\ No newline at end of file
+```
+
+[openshift]:https://www.redhat.com/en/technologies/cloud-computing/openshift
```

Comment:
these lines don't show up in the preview of the file. Need to add ``` at the end of this section.

Reply:
@zohiba I received feedback that having a tutorial and example operator link was confusing and causing users to initially navigate to the example operator instead of the tutorial. And once they were viewing the example, they were unsure what to do next.....which the goal is for users to walk through the tutorial to execute the steps to deploy this operator.

Reply:
@zohiba These are used as reference links, which is currently referenced in the prerequisite.